features/admin-file-attributes.feature

```diff
@@ -0,0 +1,142 @@
+    As a curator,
+    I want to manage file attributes from the update form
+    So that I can associate various attributes to files
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+    And an admin user exists
+
+  @ok
+  Scenario: Guest user cannot visit admin file update page
+    Given I am not logged in to Gigadb web site
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
+
+  @ok
+  Scenario: Go to a published dataset found in production-like database
+    Given I am not logged in to Gigadb web site
+    When I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    Then I should see "History" tab with text "File Termitomyces_assembly_v1.0.fa.gz updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @Published
+  Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons
+    Given I sign in as an admin
+    When I am on "/adminFile/update/id/13973"
+    Then I should see a button "New Attribute"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Edit"
+    And I should see a button input "Delete"
+    And I should see a button input "Save"
+
+  @ok @javascript @Published
+  Scenario:  Go to a published dataset found in production-like database, should see a last modified attribute
+    Given I sign in as an admin
+    And  I go to "/adminFile/update/id/13973"
+    And I should see a file attribute table
+    | Attribute Name | Value     | Unit |
+    | last_modified  | 2013-7-15 |      |
+    When I go to "/adminFile/view/id/13973"
+    Then I should see field "File Attribute Id" with value "1 "
+
+  @ok @javascript @Published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute and save
+    Given I sign in as an admin
+    And  I go to "/adminFile/update/id/13973"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    When I press "Delete"
+    Then I should not see "last_modified"
+    And I should not see "2013-7-15"
+    And I should not see a button "Delete"
+    And I press "Save"
+    And I go to "/adminFile/view/id/13973"
+    And I should see field "File Attribute Id" with empty value
```

Comment:
@kencho51 I would move the last 3 steps into a new scenario that checks the `Save` commit of deleting the file attribute, otherwise, you have observed outcomes which are followed by the press "Save" key action.

Also, the website automatically goes to "/adminFile/view/id/13973" after pressing "Save"? You could just check that you are on "/adminFile/view/id/13973" if you wait a couple of seconds as part of this scenario instead of explicitly going to the page.